test-it/karma/fixtures/propertynode/PropertyNodeReadUnwiredError.via

```diff
@@ -0,0 +1,55 @@
+define (%3AWeb%20Server%3AInteractive%3AWebApp%3AMain%2Egviweb dv(.VirtualInstrument (

```

Comment:
Discussed with Memo in person. We should prefer writing integration/acceptance tests in G instead.